guacamole/src/main/webapp/layouts/fr-fr-azerty.json

```diff
@@ -151,50 +198,56 @@
+        ],
+        "*" : [
+            { "title" : "*", "requires" : [         ] },
+            { "title" : "µ", "requires" : [ "shift" ] }
+        ],
+        "<" : [
+            { "title" : "<", "requires" : [         ] },
+            { "title" : ">", "requires" : [ "shift" ] }
+        ],	
+        "," : [
+            { "title" : ",", "requires" : [         ] },
             { "title" : "?", "requires" : [ "shift" ] }
         ],
-        "[" : [
-            { "title" : "[", "requires" : [         ] },
-            { "title" : "{", "requires" : [ "shift" ] }
-        ],
-        "]" : [
-            { "title" : "]", "requires" : [         ] },
-            { "title" : "}", "requires" : [ "shift" ] }
-        ],
-        "\\" : [
-            { "title" : "\\", "requires" : [         ] },
-            { "title" : "|",  "requires" : [ "shift" ] }
-        ],
         ";" : [
             { "title" : ";", "requires" : [         ] },
-            { "title" : ":", "requires" : [ "shift" ] }
+            { "title" : ".", "requires" : [ "shift" ] }
         ],
-        "'" : [
-            { "title" : "'",  "requires" : [         ] },
-            { "title" : "\"", "requires" : [ "shift" ] }
+        ":" : [
+            { "title" : ":",  "requires" : [         ] },
+            { "title" : "/", "requires" : [ "shift" ] }
+        ],
+        "!" : [
+            { "title" : "!",  "requires" : [         ] },
+            { "title" : "§", "requires" : [ "shift" ] }
         ],
 
-        "q" : [
-            { "title" : "q", "requires" : [                 ] },
-            { "title" : "Q", "requires" : [ "caps"          ] },
-            { "title" : "Q", "requires" : [ "shift"         ] },
-            { "title" : "q", "requires" : [ "caps", "shift" ] }
+        "a" : [
+            { "title" : "a", "requires" : [                 ] },
+            { "title" : "A", "requires" : [ "caps"          ] },
+            { "title" : "A", "requires" : [ "shift"         ] },
+            { "title" : "a", "requires" : [ "caps", "shift" ] }
         ],
-        "w" : [
-            { "title" : "w", "requires" : [                 ] },
-            { "title" : "W", "requires" : [ "caps"          ] },
-            { "title" : "W", "requires" : [ "shift"         ] },
-            { "title" : "w", "requires" : [ "caps", "shift" ] }
+        "z" : [
+            { "title" : "z", "requires" : [                 ] },
+            { "title" : "Z", "requires" : [ "caps"          ] },
+            { "title" : "Z", "requires" : [ "shift"         ] },
+            { "title" : "z", "requires" : [ "caps", "shift" ] }
         ],
         "e" : [
             { "title" : "e", "requires" : [                 ] },
             { "title" : "E", "requires" : [ "caps"          ] },
             { "title" : "E", "requires" : [ "shift"         ] },
-            { "title" : "€", "requires" : [ "caps", "shift" ] }
+			{ "title" : "e", "requires" : [ "caps", "shift" ] },
```

Comment:
Please do not include tab characters. Guacamole source files should use spaces only (4-space indent). See the surrounding lines.